ru/lessons/basics/enum.md

```diff
@@ -1,25 +1,46 @@
 ---
-version: 0.9.0
+version: 1.2.0
 layout: page
 title: Enum
 category: basics
 order: 3
 lang: ru
 ---
 
-Набор алгоритмов для операций с коллекциями.
+Набор алгоритмов для операций с перечислениями.
```

Comment:
When I first time read this lesson in english, saw that `Enum` is used in the whole text as an enumeration. I've chosen `перечисление`, because it's shorter than `перечисляемая коллекция` or `перечисляемая последовательность` which have same sense here.
What version do you prefer?